source/beamline/BioXAS/BioXASSideTableCoordinator.cpp

```diff
@@ -301,10 +284,10 @@ void BioXASSideTableCoordinator::onPitchControlValueChanged(double angle)
 
 void BioXASSideTableCoordinator::onPitchStopControlValueChanged()
 {
-	if(!connectedOnce_)
+	if(!initialized_)
 		return;
 
-	if (softIOCPitchStopControl_->withinTolerance(1)) {
+	if (softIOCPitchStatusControl_->isMoving()) {
```

Comment:
Forgot to test the stop function and it seems there is sth wrong with stop PV / logic. I will retest it.